requirements.txt

```diff
@@ -1,14 +1,7 @@
-charset-normalizer==2.0.12
-configobj==5.0.6
-exif==1.4.2
-filedate==2.0
-idna==3.3
-numpy==1.22.3
-pandas==1.4.2
-plum-py==0.8.5
-python-dateutil==2.8.2
-pytz==2022.1
-requests==2.31.0
-six==1.16.0
-swat==1.6.1
-urllib3==1.26.18
+configobj
+filedate
+numpy
+pandas
```

Comment:
Might be worth making a note about Pandas v2.1.2+ possibly not be fully supported by this project, until the way that it handles NaN values is fixed.
https://gitlab.sas.com/snztst/viyaauthmodel/-/issues/10
https://github.com/pandas-dev/pandas/issues/56379

Reply:
I will do that